common/i18n/i18nFactory.ts

```diff
@@ -53,10 +55,7 @@ export const createParams = (lang: string) => ({
   },
 })
 
-export const createI18nInstance = (lang: any, callback: any) =>
-  createInstance(createParams(lang), (_err: any, t: any) => callback({ language: lang, t }))
-
-export const createI18nPromise = (lang: any) =>
+export const createI18nPromise = (lang: any): Promise<{ language: Lang; t: TFunction }> =>
```

Comment:
return Promise<i18n>

Reply:
should i change still or leave as is? this error is when changed to return type `Promise<i18n>`